network/cache/rcvcache.go

```diff
@@ -0,0 +1,44 @@
+package netcache
+
+import (
+	"github.com/rs/zerolog"
+
+	"github.com/onflow/flow-go/model/flow"
+	herocache "github.com/onflow/flow-go/module/mempool/herocache/backdata"
+	"github.com/onflow/flow-go/module/mempool/herocache/backdata/heropool"
+	"github.com/onflow/flow-go/module/mempool/stdmap"
+)
+
+// RcvCache implements an LRU cache of the received eventIDs that delivered to their engines
+type RcvCache struct {
```

Comment:
nit: maybe rename it to `ReceiveCache` to be easy for anyone new to the codebase

Reply:
https://github.com/onflow/flow-go/pull/1908/commits/acb7563b456c7e30fbb02ac552bf6565d2b04ad7